plugins/modules/system/puppet.py

```diff
@@ -54,10 +54,11 @@
   logdest:
     description:
     - Where the puppet logs should go, if puppet apply is being used.
-    - C(all) will go to both C(stdout) and C(syslog).
+    - C(all) will go to both C(console) and C(syslog).
+    - C(stdout) is deprecated and has been replaced by C(console) since 3.0.1.
```

Comment:
```suggestion
    - C(stdout) is deprecated and has been replaced by C(console) since community.general 3.1.0.
```
Never mention version numbers without saying to what they belong :) Also deprecations cannot be made in bugfix releases.